client/client.go

```diff
@@ -145,6 +158,37 @@ func New(ctx context.Context, address string, opts ...ClientOpt) (*Client, error
 		gopts = append(gopts, grpc.WithStreamInterceptor(grpc_middleware.ChainStreamClient(stream...)))
 	}
 
+	kaTimeDuration := defaultClientKeepaliveTime
+	if timeEnv := os.Getenv("OKTETO_KEEPALIVE_CLIENT_TIME_MS"); timeEnv != "" {
+		timeDuration, err := time.ParseDuration(fmt.Sprintf("%sms", timeEnv))
+		if err != nil {
+			kaTimeDuration = defaultClientKeepaliveTime
+		}
+
+		kaTimeDuration = timeDuration
```

Comment:
```suggestion
		if err == nil {
			kaTimeDuration = timeDuration
		}
```

Reply:
I am not sure what can contain a useful log for us in this case. Any idea?